File: bridge/src/other/enums.py
from enum import Enum


class SocketEvent(str, Enum):
    START = "start"
    PROCESS = "process"
    FINISH = "finish"


class QueueType(str, Enum):
    TRANSLATION = "translation"
    TRANSCRIPTION = "transcription"
    SYNTHESIS = "synthesis"


class InputLanguage(str, Enum):
    EN = "en"
    RU = "ru"


class OutputLanguage(str, Enum):
    AZ = "az"
    TR = "tr"
    RU = "ru"

File: web/src/other/enums.ts
export enum SocketEvent {
    // basic events
    CONNECT = "connect",
    DISCONNECT = "disconnect",
    ENTER = "enter",
    EXIT = "exit",
    START = "start",
    FINISH = "finish",
    // specific events
    TRANSLATION = "translation",
    NOTIFICATION = "notification",
}

export enum ThemeTypes {
    DARK_THEME = 'DARK',
    LIGHT_THEME = 'LIGHT',
}